envconfig/config.go

```diff
@@ -215,6 +219,23 @@ func LoadConfig() {
 		}
 	}
 
+	if dlp := clean("OLLAMA_DOWNLOAD_CONN"); dlp != "" {
+		const minDownloadConnections = 1
+		const maxDownloadConnections = 64
```

Comment:
```suggestion
		const maxDownloadConnections = 1000
```
Some ollama users have really really fast (multi-gigabit) networks, let them download many parts at once